src/test/java/com/code_intelligence/jazzer/mutation/engine/SeededPseudoRandomTest.java

```diff
@@ -0,0 +1,114 @@
+
+  static Stream<Arguments> doubleClosedRange() {
+    return Stream.of(arguments(Double.NEGATIVE_INFINITY, Double.POSITIVE_INFINITY, false),
+        arguments(Double.MAX_VALUE, Double.POSITIVE_INFINITY, false),
+        arguments(Double.NEGATIVE_INFINITY, -Double.MAX_VALUE, false),
+        arguments(-Double.MAX_VALUE, Double.MAX_VALUE, false),
+        arguments(-Double.MAX_VALUE, -Double.MAX_VALUE, false),
+        arguments(-Double.MAX_VALUE * 0.5, Double.MAX_VALUE * 0.5, false),
+        arguments(-Double.MAX_VALUE * 0.5, Math.nextUp(Double.MAX_VALUE * 0.5), false),
+        arguments(Double.MAX_VALUE, Double.MAX_VALUE, false),
+        arguments(-Double.MIN_VALUE, Double.MIN_VALUE, false),
+        arguments(-Double.MIN_VALUE, 0, false), arguments(0, Double.MIN_VALUE, false),
+        arguments(-Double.MAX_VALUE, 0, false), arguments(0, Double.MAX_VALUE, false),
+        arguments(1000.0, Double.MAX_VALUE, false), arguments(0, Double.POSITIVE_INFINITY, false),
+        arguments(1e200, Double.POSITIVE_INFINITY, false),
+        arguments(Double.NEGATIVE_INFINITY, -1e200, false), arguments(0.0, 1.0, false),
+        arguments(-1.0, 1.0, false), arguments(-1e300, 1e300, false),
+        arguments(0.0, 0.0 + Double.MIN_VALUE, false),
+        arguments(-Double.MAX_VALUE, -Double.MAX_VALUE + 1e292, false),
+        arguments(-Double.NaN, 0.0, true), arguments(0.0, Double.NaN, true),
+        arguments(Double.NaN, Double.NaN, true));
+  }
+
+  static Stream<Arguments> floatClosedRange() {
+    return Stream.of(arguments(Float.NEGATIVE_INFINITY, Float.POSITIVE_INFINITY, false),
+        arguments(Float.MAX_VALUE, Float.POSITIVE_INFINITY, false),
+        arguments(Float.NEGATIVE_INFINITY, -Float.MAX_VALUE, false),
+        arguments(-Float.MAX_VALUE, Float.MAX_VALUE, false),
+        arguments(-Float.MAX_VALUE, -Float.MAX_VALUE, false),
+        arguments(Float.MAX_VALUE, Float.MAX_VALUE, false),
+        arguments(-Float.MAX_VALUE / 2f, Float.MAX_VALUE / 2f, false),
+        arguments(-Float.MIN_VALUE, Float.MIN_VALUE, false), arguments(-Float.MIN_VALUE, 0f, false),
+        arguments(0f, Float.MIN_VALUE, false), arguments(-Float.MAX_VALUE, 0f, false),
+        arguments(0f, Float.MAX_VALUE, false), arguments(-Float.MAX_VALUE, -0f, false),
+        arguments(-0f, Float.MAX_VALUE, false), arguments(1000f, Float.MAX_VALUE, false),
+        arguments(0f, Float.POSITIVE_INFINITY, false),
+        arguments(1e38f, Float.POSITIVE_INFINITY, false),
+        arguments(Float.NEGATIVE_INFINITY, -1e38f, false), arguments(0f, 1f, false),
+        arguments(-1f, 1f, false), arguments(-1e38f, 1e38f, false),
+        arguments(0f, 0f + Float.MIN_VALUE, false),
+        arguments(-Float.MAX_VALUE, -Float.MAX_VALUE + 1e32f, false),
+        arguments(-Float.NaN, 0f, true), arguments(0f, Float.NaN, true),
+        arguments(Float.NaN, Float.NaN, true));
+  }
+
+  @BeforeEach
+  void setUp() {
+    seededPseudoRandom = new SeededPseudoRandom(1337);
```

Comment:
It would be slightly cleaner to make this a local variable in the tests instead.

Reply:
Agreed, especially when reproducing single tests by commenting out the other arguments.